components/collector/src/source_collectors/gitlab/source_up_to_dateness.py

```diff
@@ -80,7 +80,8 @@ class GitLabPipelineUpToDateness(TimePassedCollector, GitLabPipelineBase):
 
     async def _parse_source_response_date_time(self, response: Response) -> datetime:
         """Override to get the date and time of the pipeline."""
-        datetimes = [self._datetime(pipeline) for pipeline in await self._pipelines(response)]
+        pipelines = await self._pipelines(SourceResponses(responses=[response]))
+        datetimes = [pipeline.datetime for pipeline in pipelines]
```

Comment:
Is the `._datetime()` then still in use?

Reply:
No, and it's removed.